framework/db/Command.php

```diff
@@ -537,6 +541,7 @@ public function createTable($table, $columns, $options = null)
     public function renameTable($table, $newName)
     {
         $sql = $this->db->getQueryBuilder()->renameTable($table, $newName);
+        $this->requireTableSchemaRefreshment($table);
```

Comment:
Imagine following:
```php
$db = Yii::$app->db;
$command = $db->createCommand()->dropTable('foo'); // prepare
// some manipulate 'foo' table data
// ...
$command->execute(); // actual execution.
```

Or:
```php
$db = Yii::$app->db;
$dropCommand = $db->createCommand()->dropTable('tmp');
if ($db->schema->getTableSchema('tmp')) {
    $dropCommand->execute(); // drop 1
}

$db->createCommand()->createTable('tmp', [...]);
// manipulate table 'tmp'

$dropCommand->execute(); // drop 2
```

Current implementation ensures cache invalidated in all cases, while simplifying it will produce unexpected results.